docs/redirects.data

```diff
@@ -0,0 +1,272 @@
+[
```

Comment:
I put all of the redirects into one file to keep it localized. I resorted to some anonymous function trickery because I was depending on some paths defined in conf, and had to add the docs directory to the pythonpath since a relative import wouldn't work.